src/notification.c

```diff
@@ -409,17 +408,13 @@ on_window_map (GtkWidget  *widget, GdkEvent   *event, WindowData *windata) {
 
 void on_size_allocate(GtkWidget *widget, GtkAllocation *allocation, WindowData *windata) {
     Settings settings = windata->settings;
-    if(settings.pos_x > 0 || settings.pos_y > 0) {
-        if (settings.center){
-            gint rwidth, rheight, depth;
-            GdkWindow *root = gtk_widget_get_root_window (GTK_WIDGET (windata->win));
-            gdk_window_get_geometry (root, NULL, NULL, &rwidth, &rheight, &depth);
-            gtk_window_move (windata->win, (rwidth - allocation->width)/2 + settings.pos_x, (rheight - allocation->height)/2 + settings.pos_y);
-        } else {
-            gtk_window_move(GTK_WINDOW(windata->win), settings.pos_x, settings.pos_y);
-        }
+    if (settings.center){
+        gint rwidth, rheight, depth;
+        GdkWindow *root = gtk_widget_get_root_window (GTK_WIDGET (windata->win));
+        gdk_window_get_geometry (root, NULL, NULL, &rwidth, &rheight, &depth);
+        gtk_window_move (windata->win, (rwidth - allocation->width)/2 + settings.pos_x, (rheight - allocation->height)/2 + settings.pos_y);
     } else {
-        gtk_window_set_position(GTK_WINDOW(windata->win), GTK_WIN_POS_CENTER);
+        gtk_window_move(GTK_WINDOW(windata->win), settings.pos_x, settings.pos_y);
```

Comment:
What happens if one, or both, of the positions have negative values for this absolute mode? Does it fail or does it render outside the screen?

Reply:
It does go outside the screen.

Reply:
![image](https://user-images.githubusercontent.com/32205463/223200627-4d6bf549-d2f4-45c4-b09c-f7418f8ba92c.png)